style/homepage.css

```diff
@@ -61,6 +61,36 @@
     color: var(--royalblue-bg);
     font-weight: 200;
 }
+.aside {
+    position: fixed;
+}
+.product-card-container {
+    height: 18.7rem;
```

Comment:
giving fix height or width might create issues with responsiveness. try to find a better way to achieve what you want